conf/test_full_somatic.config

```diff
@@ -0,0 +1,27 @@
+/*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    Nextflow config file for running full-size tests
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    Defines input files and everything required to run a full size pipeline test.
+
+    Use as follows:
+        nextflow run nf-core/sarek -profile test_full_somatic,<docker/singularity> --outdir <OUTDIR>
+
+----------------------------------------------------------------------------------------
+*/
+
+params {
+    config_profile_name        = 'Full test profile for somatic VC'
+    config_profile_description = 'Full test dataset to check somatic VC pipeline function'
+
+    // Input data for full size test
+    input = 'https://raw.githubusercontent.com/nf-core/test-datasets/sarek/testdata/csv/HCC1395_WXS_somatic_full_test.csv'
+
+    // Other params
+    tools = 'strelka,mutect2,deepvariant,freebayes,ascat,manta,vep'
```

Comment:
```suggestion
    tools = 'strelka,mutect2,freebayes,ascat,manta,vep'
```

Reply:
should we test all possible tools here? So also msisensorpro, cnvkit and controlfreec?

Reply:
should hopefully display correctly now https://nf-co.re/sarek/dev